### ISSUE TITLE
Fix tests

### ISSUE THREAD

**Issue by mtreca** [emacsorphanage/gnuplot#70]
Running `make -f Makefile.dst test` for testing fails with error:

```
In toplevel form:
gnuplot-test-context.el:5:1:Error: Symbol’s value as variable is void: gnuplot-token
make: *** [Makefile.dst:21: gnuplot-test-context.elc] Error 1
```

**Comment by mtreca** [comment]
5fd785c fixes the test runner. However, some tests are now failing.

**Comment by mtreca** [comment]
Fixed in 1dd661e
